manifests/vm-import-operator/v0.0.1/vm-import-operator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,221 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: vm-import-operator.0.0.1
+  namespace: kubevirt-hyperconverged
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Represents a virual machine import
```

Comment:
s/virual/virtual

Reply:
interesting - seems the culprit is pkg/operator/resources/operator/operator.go in which we have:
```
 DisplayName: "Virual Machine import",
```
and as a result any generated file will carry that mistake. I'll fix the origin + generated resources.